charts/vector/templates/daemonset.yaml

```diff
@@ -17,7 +17,20 @@ spec:
     metadata:
       annotations:
       {{- if .Values.rollWorkload }}
+        {{- if .Values.existingConfig }}
+          {{- range $file, $contents := (lookup "v1" "ConfigMap" (print .Release.Namespace) (print .Values.existingConfig)).data }}
```

Comment:
`lookup` fetches a resource via the kubernetes api, and we range through the `data` key which should contain config files like:

```
data:
  myConfig.yaml: |
....
```